test/fixtures/env.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   BEEP: false,
   BOOP: [ 'some', 'thing', 'that', 'goes', 'wow' ],
   BLEEP: 'false',
-  PING: [ 'ping', 'true', 2, 100 ]
+  PING: [ 'ping', 'true', 2, 100 ],
+  PONG: 'some,thing,that,goes,wow'
```

Comment:
@rodw1995 ah you're right.  the actual bug is due to how `dotenv` works.  can you please make it so that the user has to wrap the string in double quotes?  e.g. `PONG=""foo,bar,baz""`?  This is the only way it'd work.  Can you update the Readme and source code based off this double quote concept?  Thank you!